templates/views/layouts/default.html

```diff
@@ -12,19 +12,19 @@
 
     <!-- STYLESHEETS -->
     {# Customise the stylesheet for your site by editing /public/styles/site.sass #}
-    <link href="/styles/site.css" rel="stylesheet">
+    <link href="../styles/site.css" rel="stylesheet">
     <!-- CUSTOM STYLES -->
-    <link rel="stylesheet" type="text/css" href="styles/variables.css">
-    <link rel="stylesheet" type="text/css" href="styles/main.css">
-    <link rel="stylesheet" type="text/css" href="styles/index.css">
+    <link rel="stylesheet" type="text/css" href="../styles/variables.css">
+    <link rel="stylesheet" type="text/css" href="../styles/main.css">
+    <link rel="stylesheet" type="text/css" href="../styles/index.css">
```

Comment:
Just curious why a lot of paths were updated with `..` like this. Did the former stop working?

Reply:
This is so because of the new route level. Without the `..` , the links can't be seen by the /events/ev.. pages. None of the stylesheets or links will work at that level.